database/migrations/default/1729527603827_recreate_crashes_list_view/up.sql

```diff
@@ -0,0 +1,100 @@
+-- Recreating this view and removing references to crashes_edits
+
+CREATE OR REPLACE VIEW public.crashes_list_view
+AS WITH geocode_status AS (
+         SELECT cris.id,
+            unified.latitude IS NOT NULL OR unified.longitude IS NOT NULL AS is_manual_geocode
+           FROM crashes_cris cris
+             LEFT JOIN crashes unified ON cris.id = unified.id
+        )
```

Comment:
for `is_manual_geocode`, this needs to be changed to check if the `unified` value is different from the `cris` value—in which case it should be `true`.

Reply:
:heavy_check_mark:  thank you!